script/Ciel/Day 6/06_01_CIEL06_1.txt

```diff
@@ -211,7 +211,8 @@ Resisting the temptation to abandon this train of thought, I take ten seconds to
 [sha:8d39d8369ea04082a441a2c956f9f0981fc07319]{
 -- Page 14, Offset 13189.
 -- ……どう考えても、この二日間の行動は弁明のしようがない。無断欠席。門限破り。おまけに音信不通ときた。
-No matter how much I rack my brain, I can't think of any way I could explain the events of the past two days...
+// Minor nitpick, but the phrasing makes it sound like absence, curfew, etc are explanations, rather than events. Threw in a rewrite to explain what I mean, thoughts/revisions?
+No matter how much I rack my brain, the events of the last two days seem impossible to explain...
```

Comment:
This changes the meaning a bit I think - it's not that the events can't be explained, it's that he can't explain _himself_. Maybe 
> [...] I can't work out how to explain away the events of the past two days...

Reply:
I feel like using 'events' itself causes that slight shift in nuance. Making the subject himself like in 'I can't explain what I was up to in the past two days,' is probably more appropriate.